locales/ar-arb/deliverytool.rdf.po

```diff
@@ -1,7 +1,7 @@
 msgid ""
 msgstr ""
-"Project-Id-Version: TAO 3.4.0-sprint132\n"
-"PO-Revision-Date: 2020-07-17T11:49:30\n"
+"Project-Id-Version: TAO 2023.12\n"
```

Comment:
Thanks, I am just not sure about those changes. If it will impact translations we have. As you can see, Sprint 132 is almost 4 years old. I am not sure we are 4 years without adding translations in this extension. 

@jsconan Do you think there is any risk on this?